gamemodes/terrortown/gamemode/roles/clown/shared.lua

```diff
@@ -12,6 +12,8 @@ CreateConVar("ttt_clown_show_target_icon", "0", FCVAR_REPLICATED)
 CreateConVar("ttt_clown_heal_on_activate", "0", FCVAR_REPLICATED)
 CreateConVar("ttt_clown_heal_bonus", "0", FCVAR_REPLICATED, "The amount of bonus health to give the clown if they are healed when they are activated", 0, 100)
 CreateConVar("ttt_clown_damage_bonus", "0", FCVAR_REPLICATED, "Damage bonus that the clown has after being activated (e.g. 0.5 = 50% more damage)", 0, 1)
+CreateConVar("ttt_clown_can_see_jesters", 0, FCVAR_REPLICATED)
+CreateConVar("ttt_clown_update_scoreboard", 0, FCVAR_REPLICATED)
```

Comment:
I think it makes more sense here to have these be enabled by default to match the precedent set by other independent roles with conditions to be the last man standing. These also need to be added to CONVARS.md
```suggestion
CreateConVar("ttt_clown_can_see_jesters", 1, FCVAR_REPLICATED)
CreateConVar("ttt_clown_update_scoreboard", 1, FCVAR_REPLICATED)
```